app/components/DataTable/index.scss

```diff
@@ -17,6 +17,20 @@
     .ag-row:last-child {
       border-bottom: none;
     }
+
+    .cell-span, .cell-span--last {
```

Comment:
is this all just because you want it centered vertically vs the default top aligned or was there other issues with the borders?

Reply:
both actually, there was the issue where the rowSpanned cell was not actually the last row according to the table, so there would be a double border at the visual bottom of the table